.gitignore

```diff
@@ -133,3 +133,7 @@ src/
 logs/*.log
 
 # output files: local gitignore added to city_scrapers/local_outputs/
+
```

Comment:
Usually we'll want to keep changes like this separate from a scraper PR. You can add it to `.git/info/exclude` to exclude those files in your local repo though without changing the .gitignore

Reply:
>Usually we'll want to keep changes like this separate from a scraper PR. 
Sure, that's something how I should have cleaned up:)